### ISSUE TITLE
Swipe from left bug

### ISSUE THREAD

**Issue by TadeasKriz** [hackiftekhar/IQKeyboardManager#464]
When I click into a UITextField and then start swiping right from left edge, but only for about 6th of the screen then then put it back (so that I stay in the controller I'm in), it often results in somehow wrongly positioned view. See the following screenshot:

On iPhone 6s - iOS 9.2
![iOS 9.2](https://lh3.googleusercontent.com/dUN5Pfm-27xgYc5ZjuqlajcmC2bpeDfCQcR4LCIdSKA5w7FJitbhzvgCV3L7M0BmfusW35JozDCF0nYWENL0fSWFiaBucmPVYl3kgH4siFmAgDjCmZIqu8_YnzG4fyfiMRLS4XLbc9Gnp4V5jMmVxlYqOluTbOjxU4UGrg00BpnMwdvKq2Zq8e0cYgE93gFfQd_sZdZfrcosxk6Rt_vmZIsfnNO9WBPPjhVIRbrEMoYMHPHTSfMOLU_mu20UFCwRwE8CqmekwCiMiz7bTOehGr-25gNKAgJaQYj47M6tDPpGE7OHI-NM9GUnXuLjBVLTCcqbnrSxGI7DLtH4CfAIKF4P0Apo4_1Hi6qEk9hmEPctnldip7mssRpoUaJf6ZuEtOjAqu8zDlDSu62rQo2-bd1eFbhT1I-29u4ouoir34e3HBlHg_h3lNL6RGpd55Tc55HQNReAiAdA6ysVl5ZGFu3ZvQvRB4NRfqqNo81YT1j5qWPilV_32idp9M72Y5wa1-1hNDFGb9eMEeXt2xG3xH1bX1RUHl9ugtTy_zf6XEIPolH_OEh46y-p3FlQUt0_BhEW7Q=w750-h1334-no)

On iPhone 5 - iOS 8.4
![iOS 8.4](https://lh3.googleusercontent.com/ypvisli0BC5LvxC4TkrUaxaC-Em0An6hEC26-VPPnpKTrpIyDPJQV9SGlczE1tuAO2XKGQxUE83QLFAQvWxMueCUNg17vRHVJBRh-6c0Zt9ZjnBsyKsmogT9vE09eIpBiMHnOG9V9xnEHlr0IJtJU727vwMrdLXi9s8dx7MmbyD8wN1kodR8ORY5EDIgqqnLNaTMGu937okEqy1VFBJsiWulVje695e983kKEg1atfCDRT4bUNI6itzMXNfyZcvCXFnT6NGxAubgRgulDDDbDkW4YAETHg5umeY1jlexy52DlY5hDa0zu10dt8yFlDu7qyI2NthxcJ-uqQGFKTbKuCqz42EdVKHKstfIl65wUE2pwZjDK1RGct60GGU6l-sQShQ3uAT1RhJqPtHqeeMk8_B8oXmMCpKq5XKryVWkHQGhFJldLy49tS1YDxgtu_gPBskDmjHWHeYubjxEoStNJzI3tFtKtTKAg1fw8Mq5n2-KcrpDe4oLmj8eVMDkdXE0eldiL_HAvLhiY-bT9h1rrmfmEHsK0H1QsZLrRain4hohhgyoaHCpxXbyfzVdlQ-JU3ltig=w640-h1136-no)

**Comment by hackiftekhar** [comment]
And you are sure that the textField is resigned and this is not the one from Xcode Simulator Known Issue explained in README. Correct?

**Comment by hackiftekhar** [comment]
And you are sure that the textField is resigned and this is not the one from Xcode Simulator Known Issue explained in README. Correct?

**Comment by TadeasKriz** [comment]
This happens in the demo app as well as in my own. I'm not sure if the demo app does resign it. I though the known issue is talking about black toolbar, not this black space. I'll try to add resignation to see.

**Comment by hackiftekhar** [comment]
Are you sure this issue is reproducible with Real iOS devices?

**Comment by TadeasKriz** [comment]
Yes, I found it on my iPhone 6 with iOS 9.3.

**Comment by hackiftekhar** [comment]
Ok, then could you please provide me a video?

**Comment by TadeasKriz** [comment]
I'll try.

**Comment by TadeasKriz** [comment]
https://www.youtube.com/watch?v=uDlAQLDBdm0

**Comment by hackiftekhar** [comment]
Thank You! I'll try to take a look at it tomorrow.

**Comment by TadeasKriz** [comment]
@hackiftekhar Thanks!

**Comment by TadeasKriz** [comment]
Do you think it is related to the known bug?

**Comment by hackiftekhar** [comment]
If this is reproducible with real device then it's not from the known but and it's new bug. I can comment something after looking the video.

**Comment by hackiftekhar** [comment]
I checked and I found that it's a bug. We need to fix it.

**Comment by hackiftekhar** [comment]
I checked the code and I found that IQKeyboardManager is doing correct frame changes, but iOS reset the ViewController's frame in the middle somewhere. I tried to capture frame changes using observers. But observer also doesn't capture the frame change made by iOS (with pop swipe gesture). I'm sure it's related to iOS, but still we need to do some workaround to avoid this issue. I feels that it's hard to fix because all the keyboard and textfield notifications are coming in exact order like a normal textField does with begin/resign responder. I don't have any clues by which I could identify something to fix this situation.
